src/Collections/Sites.php

```diff
@@ -261,33 +287,40 @@ protected function getUuidByName(string $name): string
     }
 
     /**
-     * Looks up a site's UUID by its name.
+     * Returns the site UUID.
      *
-     * @param string $id Name of the site to look up
+     * @param string $site_id
+     *   The site name or UUID.
      * @return string
+     *   The site UUID.
+     *
+     * @throws \Pantheon\Terminus\Exceptions\TerminusException
+     * @throws \Pantheon\Terminus\Exceptions\TerminusNotFoundException
+     * @throws \GuzzleHttp\Exception\GuzzleException
      */
-    protected function findUUIDByNameOrUUID($id): ?string
+    protected function getUuidBySiteId(string $site_id): ?string
```

Comment:
The new name isn't as clear as the old one. The docblock still stipulates "name or uuid", but this is less clear when reading the code. It's protected & no one should be extending sites, but the old name seemed fine. I'm indifferent between get and find (and camelcase conventions), but maybe keep 'NameOrUuid'?